generated/nifpga/nifpga_service.cpp

```diff
@@ -1078,12 +1076,10 @@ ::grpc::Status MonikerWriteArrayDbl(void* data, google::protobuf::Arena& arena,
     auto session = function_data->session;
     auto control = function_data->control;
 
-    ArrayDoubleData arraydoubledata_message;
-    packedData.UnpackTo(&arraydoubledata_message);
-    
-    auto data_array = arraydoubledata_message.value();
-    auto array = const_cast<double*>(arraydoubledata_message.value().data());
-    auto size = data_array.size();
+    MonikerWriteArrayDblRequest* request;
+    packedData.UnpackTo(request);
+      auto array = const_cast<double*>(request->array().data());
```

Comment:
There are a lot of formatting issues in this file, you should fix that.

Reply:
This will be taken care as part of a follow up cleanup PR with other things.